.circleci/config.yml

```diff
@@ -66,7 +66,7 @@ commands:
             python .circleci/conda_config.py "$(python setup.py --version)"
             cd evalml-core-feedstock
             echo "$DOCKERHUB_PASSWORD" | docker login -u $DOCKERHUB_USERNAME --password-stdin
-            export DOCKER_CONTAINERID="$(docker run -td condaforge/linux-anvil-comp7)"
+            export DOCKER_CONTAINERID="$(docker run -td condaforge/linux-anvil-cos7-x86_64)"
```

Comment:
Yes, we need to change the docker image we use in our conda-check because because `python-kaleido` requires a centOS docker image. 

This reflects this [change](https://github.com/conda-forge/evalml-core-feedstock/pull/17) we had to make to our conda recipe.

I tested this change locally by checking out this branch and running our [conda build steps](https://github.com/alteryx/evalml/blob/main/.circleci/config.yml#L59) manually. Tests pass and the build succeeds!